packages/website/ts/blockchain.ts

```diff
@@ -65,7 +65,7 @@ export class Blockchain {
     public nodeVersion: string;
     private zeroEx: ZeroEx;
     private dispatcher: Dispatcher;
-    private web3Wrapper: Web3Wrapper;
+    private web3Wrapper?: Web3Wrapper;
```

Comment:
Um, why is this now optional?

Reply:
It always was. Do we set it in the constructor? No. Then it's optional. It's better to have it as a type so that if we ever change the code - we're more carefull